src/pages/main/HomePage/components/LnbList.tsx

```diff
@@ -0,0 +1,86 @@
+import { useState } from 'react';
+import styled, { css } from 'styled-components';
+
+import { LnbInfo } from '../constants';
+
+const LnbList = () => {
+  const [isHoverIcon, setIsHoverIcon] = useState<boolean[]>([false, false, false, false, false, false, false, false]);
+  const [isClickIcon, setIsClickIcon] = useState<boolean[]>([false, false, false, false, false, false, false, false]);
```

Comment:
타입 지정 잘했네요 !
이 부분은 조금 더 간결하게 이렇게 바꿔볼 수 있어요!
```suggestion
  const [isHoverIcon, setIsHoverIcon] = useState<boolean[]>(Array(8).fill(false));
const [isClickIcon, setIsClickIcon] = useState<boolean[]>(Array(8).fill(false));
```
es6 문법 많이 사용하기!

Reply:
우와 이건 저도 배워갑니다!! 이렇게 작성하면 깔끔하게 나타낼 수 있겠네요!! es6 문법 많이 사용하기... 메모

Reply:
오 저는 이런 부분 원래 Array.from({length: 8}, ()=>false) 이렇게 짜는데 Array.fill을 사용하는게 더 간결하네요! 배워갑니다!!!!

Reply:
오,,, 이렇게 하면 조금 더 컴팩트하게 작성할 수 있겠군요,,, 하나 배우고 갑니다